packages/komencikit/src/verifyWallet.ts

```diff
@@ -2,32 +2,26 @@ import { Address, normalizeAddress } from '@celo/base'
 import { Err, Ok, Result } from '@celo/base/lib/result'
```

Comment:
This is fixed here: https://github.com/celo-org/celo-monorepo/pull/5724

Reply:
Sweet! Please ping me when it's merged